docs/content/kv_commands.md

```diff
@@ -25,3 +25,13 @@ $ wrangler kv create "new kv namespace"
     title: "new kv namespace",
 }
 ```
+
+### `delete <namespace-id>`
+
+#### Usage
+
+``` sh
+$ wrangler kv delete f7b02e7fc70443149ac906dd81ec1791
```

Comment:
So, one strange thing here: I would expect this to be "delete a key", not "delete a namespace". How do those two things look?

Reply:
well to delete a key you'd have to pass a namespace id in addition to the key. but we should talk about how these things should look!

Reply:
so
``` sh
$ wrangler kv delete f7b02e7fc70443149ac906dd81ec1791 # delete this namespace
# vs
$ wrangler kv delete f7b02e7fc70443149ac906dd81ec1791 foo # delete the key "foo"
```

Reply:
@steveklabnik with that 👍 can you give this an approving review